Headers/AppKit/NSPasteboard.h

```diff
@@ -296,14 +316,54 @@ APPKIT_EXPORT NSString *NSPasteboardCommunicationException;
 @end
 
 #endif
-//
-// Return File-related Pasteboard Types
```

Comment:
Similar to the other question: Any reason to remove this comment? I don't require that you keep it, I'm just interested whether we gain something by removing it.